test/metabase/driver/clickhouse_test.clj

```diff
@@ -175,3 +176,17 @@
    (is (= nil (#'clickhouse-qp/extract-datetime-timezone "datetime")))
    (is (= nil (#'clickhouse-qp/extract-datetime-timezone "datetime64")))
    (is (= nil (#'clickhouse-qp/extract-datetime-timezone "datetime64(3)")))))
+
+(deftest clickhouse-insert
+  (mt/test-driver
+   :clickhouse
+   (t2.with-temp/with-temp
+     [:model/Database db
+      {:engine  :clickhouse
+       :details (tx/dbdef->connection-details :clickhouse :db {:database-name "default"})}]
+     (driver/drop-table!   :clickhouse (:id db) "insert_test")
```

Comment:
I recommend using a random table name, so that concurrent runs for different branches can't conflict.

Reply:
Happy with 8eff462b75fa74784fec98eef1239a063052ab2e?

Or would you rather a (random-uuid) or something else. Was thinking of sql identifiers and dashes and I'd have to str/replace so this felt more parsimonious.

Reply:
Yeah I think this is fine, and has the added advantage of seeing how old a left over table is, e.g. if the process died before deleting it.